android-pdf-viewer/src/main/java/com/infomaniak/lib/pdfview/scroll/DefaultScrollHandle.java

```diff
@@ -1,41 +1,56 @@
 package com.infomaniak.lib.pdfview.scroll;
 
+import static android.view.ViewGroup.LayoutParams.WRAP_CONTENT;
+
 import android.content.Context;
 import android.graphics.Color;
 import android.graphics.drawable.Drawable;
 import android.os.Handler;
-import androidx.core.content.ContextCompat;
+import android.os.Looper;
 import android.util.TypedValue;
+import android.view.LayoutInflater;
 import android.view.MotionEvent;
-import android.view.ViewGroup;
+import android.view.View;
 import android.widget.RelativeLayout;
 import android.widget.TextView;
 
+import androidx.core.content.ContextCompat;
+
 import com.infomaniak.lib.pdfview.PDFView;
 import com.infomaniak.lib.pdfview.R;
+import com.infomaniak.lib.pdfview.util.TouchUtils;
 import com.infomaniak.lib.pdfview.util.Util;
 
 public class DefaultScrollHandle extends RelativeLayout implements ScrollHandle {
 
-    private final static int HANDLE_LONG = 65;
-    private final static int HANDLE_SHORT = 40;
-    private final static int DEFAULT_TEXT_SIZE = 16;
-
-    private float relativeHandlerMiddle = 0f;
+    private static final int HANDLE_WIDTH = 65;
+    private static final int HANDLE_HEIGHT = 40;
+    private static final int DEFAULT_TEXT_SIZE = 12;
+    private static final int NO_ALIGN = -1;
 
+    private final Handler handler = new Handler(Looper.getMainLooper());
+    private final boolean inverted;
+    private final Runnable hidePageScrollerRunnable = this::hide;
     protected TextView textView;
     protected Context context;
-    private boolean inverted;
+    private float relativeHandlerMiddle = 0f;
     private PDFView pdfView;
     private float currentPos;
-
-    private Handler handler = new Handler();
-    private Runnable hidePageScrollerRunnable = new Runnable() {
-        @Override
-        public void run() {
-            hide();
-        }
-    };
+    private Drawable handleBackgroundDrawable;
+    private View handleView;
+    private int handleAlign;
+    private int handleWidth = HANDLE_WIDTH;
+    private int handleHeight = HANDLE_HEIGHT;
+    private int handlePaddingLeft = 0;
+    private int handlePaddingTop = 0;
+    private int handlePaddingRight = 0;
+    private int handlePaddingBottom = 0;
+
+    private int hideHandleDelayMillis = 1000;
+
+    private boolean isTouched = false;
```

Comment:
Can't we avoid doing all of this by not setting a margin in the views and rather only using the margins inside the computations?